src/components/course/tests/CourseRunCard.test.jsx

```diff
@@ -99,8 +99,8 @@ const renderCard = ({
   // need to use router, to render component such as react-router's <Link>
   renderWithRouter(
     <AppContext.Provider value={INITIAL_APP_STATE}>
-      <SubsidyRequestsContext.Provider value={initialSubsidyRequestsState}>
-        <UserSubsidyContext.Provider value={initialUserSubsidyState}>
+      <UserSubsidyContext.Provider value={initialUserSubsidyState}>
```

Comment:
`<UserSubsidyContext>` provider should render before `SubsidyRequestsContext` provider according to the component tree structure that we have defined.